### PR TITLE
added pr procedure doc

### LINKED ISSUE #492: Make Document on PRs and Review process.

### PR DESCRIPTION
**Pull request by KhalidUmuzi**
Related issues: [please specify]

## I solemnly swear that:

- [X] My code follows the style guidelines of this project
- [X] I have merged the develop branch into my branch and fixed any merge conflicts
- [X] I have performed a self-review of my own code
- [X] I have commented my code in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pull_request_procedure.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+By ticking the points on the checklist, you are acknowledging that you have followed the description on that point. So don’t just tick the boxes.
+
+Once your PR has been opened, link it to the relevant issue and move the issue into the Review column for other Technical Staff to have a look at
+
+### Addressing feedback on your Pull Request
+When addressing an existing PR that has some feedback on it, it is vitally important to address said feedback before moving the card back into review. The important thing to remember is that its not enough to just address it in code. 
+
+For this, we will be using the github conversation as a way to provide feedback as well as show that a person has addressed the feedback. On an existing Pull Request, you will be receiving feedback on the “Conversation” tab as well as the “Files Changed” tab. If you are done addressing the feedback, you need to make the reviewer aware that you have done so. If you click the “Resolve Conversation” button, you acknowledge that you have addressed the feedback provided on that part of the conversation. PRs that don’t have all conversations resolved will be pushed back into the review column.
```
Comment (Sbonelo01) [inline pull_request_procedure.md]: Here, I think you meant:

"PRs that don’t have all conversations resolved will be pushed back into the review feedback column" ?

File: pull_request_procedure.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+If you have addressed the feedback in the form of a chat outside of github, please mention it regardless. Eg “Khalid has made a change and Sheena has provided feedback on why this piece of code is here. They have a conversation offline and agree that Khalid has done the right thing. Khalid needs to make sure that the rest of the team knows that this conversation happened.”
```
Comment (Sbonelo01) [inline pull_request_procedure.md]: Maybe change "They have a conversation offline ..." to "They had a conversation offline ..." Also, I am not sure if the last sentence should still be inside inverted commas.

File: pull_request_procedure.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+### Reviewing a Pull Request
+When reviewing a Pull request there are a couple of things to keep in mind. You need to understand the requirements of the issue first and make sure that the code solves said issue. Run the code on your local machine and test using various test cases to make sure it works.
+
+The second thing to make sure of is that all previous feedback needs to be addressed in this PR. There may be a long conversation that took place on the same PR and you need to be aware of this change
```
Comment (Sbonelo01) [inline pull_request_procedure.md]: Maybe these two points can be bullets?

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]
Slight grammar mistakes ...

**Review by Sbonelo01 — APPROVED** [review]

**Review by TheAbstract — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**